apps/Launcher/qml/DemoLauncher/RenderingResourcesManager.js

```diff
@@ -49,9 +59,13 @@ Communicator.prototype.isRunning = function() {
     return this.currentStatus === SESSION_STATUS_RUNNING;
 }
 
-Communicator.prototype.sendRequest = function(method, module, command, body, callback) {
+Communicator.prototype.sendRequest = function(method, module, command, body, callback, omitSessionId) {
     var fullUrl = this.serviceUrl + '/' + module + '/' + command;
 
+    if (!omitSessionId) {
+        fullUrl += ('?session_id=' + this.sessionId);
+    }
```

Comment:
You're probably right, but since there are more magic strings in this file (e.g. module just above) I'll just leave it as is for now.